resources/MSBuildImports/Current/Microsoft.Common.CrossTargeting.targets/ImportAfter/Microsoft.TestPlatform.CrossTargeting.targets

```diff
@@ -0,0 +1,84 @@
+<!--
+***********************************************************************************************
+Microsoft.TestPlatform.CrossTargeting.targets
```

Comment:
Are these .targets files supposed to be part of this PR?  It seems like they're not being consumed.

Reply:
They are copied into the msbuildextensions nupkg and then put into VS. But I changed things around a couple of times. It might be they are no longer needed. I will double check. Thanks for bringing it up.